README.md

```diff
@@ -349,6 +349,20 @@ hooks.waiting.tap('yourListenerName', () => {
 });
 ```
 
+## Typings
+
+To use the plugin typings, you have to install `@types/webpack`. It's not included by default to not collide with your
+existing typings (`@types/webpack` imports `@types/node`). [It's an old TypeScript issue](https://github.com/microsoft/TypeScript/issues/18588), 
+the alternative is to set `skipLibCheck: true` in the `compilerOptions` 😉
```

Comment:
Did you know that `skipLibCheck: true` is the default as of TypeScript 3.9?

https://www.twitter.com/orta/status/1260583041740812288

Reply:
Good to know 😃